src/garage/torch/algos/trpo.py

```diff
@@ -74,15 +74,18 @@ def __init__(self,
                          stop_entropy_gradient=stop_entropy_gradient,
                          entropy_method=entropy_method)
 
-    def _compute_objective(self, advantages, valids, obs, actions, rewards):
-        """Compute the surrogate objective.
+    def _compute_objective(self, advantages, obs, actions, rewards):
+        """Compute objective value.
 
         Args:
-            advantages (torch.Tensor): Expected rewards over the actions
-            valids (list[int]): length of the valid values for each path
+            advantages (torch.Tensor): Advantage value at each step.
```

Comment:
i think you mean `(N \dot [T], )` here in most cases. see the terminology in https://github.com/rlworkgroup/garage/blob/master/src/garage/_dtypes.py